airdrop/infrastructure/models.py

```diff
@@ -115,6 +117,9 @@ class UserBalanceSnapshot(Base, AuditClass):
     )
     address = Column("address", VARCHAR(50), nullable=False)
     balance = Column("balance", BIGINT, nullable=False)
+    staked = Column("staked", BIGINT, nullable=False)
+    total = Column("total", BIGINT, nullable=False)
+    snapshot_guid = Column("snapshot_guid", VARCHAR(50), nullable=False)
```

Comment:
If guid type in hex then we have to increase the limit to 68

Reply:
Its UUID so 50 is plenty